test/system/post_test.rb

```diff
@@ -247,12 +247,9 @@ def assert_page_does_not_reload(message = "page should not reload")
     find("span[data-original-title='Tools']").click()
     find("input[value='Give']").click()
 
-    # Wait for the alert to be shown
-    wait_for_ajax
```

Comment:
Removing `wait_for_ajax` here, I think in my prior days of writing system tests, `page.find` implicitly waited for AJAX updates.